src/Magnum/Timeline.cpp

```diff
@@ -61,4 +61,10 @@ Float Timeline::previousFrameTime() const {
     return duration_cast<microseconds>(_previousFrameTime-_startTime).count()/1e6f;
 }
```

Comment:
For feature parity I'm thinking `currentFrameTime()` could make sense as well; this one also appropriately handling the case of the timeline being stopped.